pkg/api/v1/pod/util.go

```diff
@@ -236,6 +237,7 @@ func VisitPodConfigmapNames(pod *v1.Pod, visitor Visitor) bool {
 	return true
 }
 
+// visitContainerConfigmapNames returns true if it finds reference to ConfigMap in the container, otherwise false
```

Comment:
This is not an accurate description of what this function is doing. This returns true unless the visitor returned false when invoked with a configmap reference.

Reply:
@liggitt thanks will be correcting it sorry for the delay was having exams